Common/Statistics/PortfolioStatistics.cs

```diff
@@ -285,7 +282,7 @@ private static decimal DrawdownPercent(SortedDictionary<DateTime, decimal> equit
         /// <returns>Double annual performance percentage</returns>
         private static decimal GetAnnualPerformance(List<double> performance, int tradingDaysPerYear = 252)
         {
-            return (decimal)performance.Average() * tradingDaysPerYear;
+            return (decimal)Math.Pow((performance.Average() + 1), tradingDaysPerYear) - 1;
```

Comment:
I'd suggest removing this method and using the existing fixed implementation at Statistics